ml3d/datasets/utils/bev_box.py

```diff
@@ -113,7 +136,9 @@ def to_camera(self):
         bbox = np.zeros((7,))
         bbox[0:3] = self.center - [0, 0, self.size[1] / 2]
```

Comment:
It may be a good idea to comment on what is happening here to simplify maintenance in the future.